website/src/Router.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { shallow, mount, render, configure} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { App } from './App';
+import { MemoryRouter } from 'react-router';
+import {
+  Home, Signin, Signup, ProtectedCreateClass, ProtectedJoinClass, NotFound,
+  ProtectedDashboardRouter,
+} from './views';
+import config from './views/firebaseconfig';
+import firebase from 'firebase/app';
+firebase.initializeApp(config);
+
+configure({ adapter: new Adapter() }); // configure of enzyme and react
+describe("Router Test", () => {
+  // home page
```

Comment:
Great test. Can you use /**/ comment instead of // comment? Also please left a new line between each section, that will be much better.